models/acl.py

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env python3
```

Comment:
\*shrug\* It's a habit of mine. Gives the viewer a constant reminder they're dealing with python3 code. We still have file permissions preventing most files from being executed like that, but if you don't like shebangs in non-executable files, I can remove them. Should I?